ext/date/config0.m4

```diff
@@ -8,15 +8,24 @@ AX_CHECK_COMPILE_FLAG([-Wno-implicit-fallthrough],
   [PHP_DATE_CFLAGS="$PHP_DATE_CFLAGS -Wno-implicit-fallthrough"],,
   [-Werror])
 
-PHP_DATE_CFLAGS="$PHP_DATE_CFLAGS -I@ext_builddir@/lib -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -DHAVE_TIMELIB_CONFIG_H=1"
+PHP_DATE_CFLAGS="$PHP_DATE_CFLAGS -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -DHAVE_TIMELIB_CONFIG_H=1"
+PHP_TIMELIB_CFLAGS="$PHP_DATE_CFLAGS"
+PHP_DATE_CFLAGS="$PHP_DATE_CFLAGS -I@ext_builddir@/lib"
+
+AX_CHECK_COMPILE_FLAG([-fwrapv],
+  [PHP_TIMELIB_CFLAGS="$PHP_TIMELIB_CFLAGS -fwrapv"],,
+  [-Werror])
```

Comment:
Wouldn't this break the build for compilers not supporting `-fwrapv`?

Reply:
That's what the `AX_CHECK_COMPILE_FLAG()` is for, I believe. :)

Reply:
Right. This is exactly what we want: fail the *test* if the flag is unsupported.

Sorry for the noise!